### DISCUSSION TITLE
Open file from review removed?

### DISCUSSION THREAD

**Discussion by cvan20191** [st3v3nmw/obsidian-spaced-repetition#916] [Q&A]
In the videos/demonstration, it showed a way to open the file on the card you are reviewing. Was this removed with the current version??

**Comment by ronzulu** [comment]
Hi @cvan20191 

It was removed some time last year, and there are definitely people who would like it returned.

See [ [FEAT]: leaving modus "review flashcards" and directly jump to note #786 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/786)
